src/datascience-ui/common/index.ts

```diff
@@ -108,7 +109,7 @@ export function formatStreamText(str: string): string {
 export function appendLineFeed(arr: string[], modifier?: (s: string) => string) {
     return arr.map((s: string, i: number) => {
         const out = modifier ? modifier(s) : s;
-        return i === arr.length - 1 ? `${out}` : `${out}\n`;
+        return i === arr.length - 1 ? `${out}` : `${out}${os.EOL}`;
```

Comment:
Not sure this is correct. I remember Jupyter used just `\n` (line feeds).
E.g. if we have some multi-line code in a non-python kernel, then it could crash (I know R kernel doesn't support CRLF & we had to ensure we didn't use that else the execution would just crash)

Reply:
I'll make it only used for export.

Reply:
Export tests were failing and the output made them look the same.

Reply:
ipynb already remove any \r\n in them. However export generates python files which assumes the same EOL as the OS. The tests actually verify this is the case.

Reply:
Oh this is export ipynb to py. i thought it was the other way around.